### ISSUE TITLE
dbpedia (and marinv) released dataids contain wrong dataid:Group IRIs

### ISSUE THREAD

**Issue by Vehnem** [dbpedia/databus-maven-plugin#52]
```
<https://databus.dbpedia.org/dbpedia/wikidata>
        a       dataid:Group .
```
http://downloads.dbpedia.org/repo/lts/wikidata/instance-types/2018.11.30/dataid.ttl

**Comment by Vehnem** [comment]
Sorry, something different is wrong. 
At https://databus.dbpedia.org/ is stumbled over the "MOST ACTIVE PUBLISHERS" field and something with the query sems to be wrong. For white_ghecko its fine but not for the rest of the entries.

**Comment by Vehnem** [comment]
Okay dataid:account sems to be wrong. 
But it was already fixed in commit 518f720a2533d252730d07f20894137632ef60cf (fixed account iri) on Feb. 21
